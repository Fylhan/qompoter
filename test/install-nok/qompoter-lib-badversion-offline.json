{
	"name": "test/install-nok-badversion-lib",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/acme-cp": "v0.1-lib",
		"cp/fylhan2": "v1.0"
	}
}
