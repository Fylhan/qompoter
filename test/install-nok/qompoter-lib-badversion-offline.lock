{
  "name": "test/install-nok-badversion-lib",
  "date": "2016-12-02T18:24:51+01:00",
  "require": {
  }
}
