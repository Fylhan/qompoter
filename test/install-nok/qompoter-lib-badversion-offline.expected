Qompoter
======== install

* cp/acme-cp v0.1-lib
  Downloading lib...
  Error: no library found "qompoter-repo/cp/acme-cp/v0.1-lib"
  FAILURE

FAILURE
